### PR TITLE
added install instructions to readme and updated faq section

### PR DESCRIPTION
**Pull request by jandetlefsen**
Added more content to the readme file and gave it a hierarchy because the sidebar nav items where getting too many and rendered itself useless imo. Most new content credit to @newnils

### CODE REVIEW

**Comment by newnils** [comment]
Thank you @jandetlefsen for featuring my FAQ :)

File: docs/getting-started/faq.md [hunk 1]
```diff
@@ -1,71 +1,73 @@
...
+„Status is an open source messaging platform and mobile interface to interact with decentralized applications that run on the Ethereum Network.” (quoted from [Whitepaper](https://status.im/whitepaper.pdf))
 
-The average user lifetime sky rockets once their friends are involved. When you start thinking about this as a user base and how to connect that with Ethereum, things like payments and applications built ontop of Ethereum integrated into a converstional interface starts becoming like a clear path forward.
+### What does Status want to solve?
 
-Ultimately we're building a hybrid browser and messenger for us to have the best chance to cast the widest net and focus on user acquisition, staying agnostic and as close as possible to the principles Ethereum embodies.
+Status wants to solve the problem that we (the users) have different identities all over the internet. It criticized that we have spreaded to many personal information all over various platforms. Status wants to create a digital hub where we can use various services based on the Ethereum Network.
```
Comment (andytudhope) [inline docs/getting-started/faq.md]: "we have spreaded to many personal information all over various platforms" should be:

"we have spread too much personal information across various platforms". 

I really appreciate this PR, but cannot merge it with grammar and spelling errors like this, sorry.

File: docs/getting-started/faq.md [hunk 2]
```diff
@@ -1,71 +1,73 @@
...
+When Carl & Jarrad looked at how to achieve mass adoption for a client. We looked at things such as where the people are, how they behave on their devices, time spent on pc vs mobile - as of 2014 more time is spent on mobile than desktop, and of that time - a THIRD of it is spent within Instant Messengers.
 
- 
-## Can your app be launched with a link?
+Instant Messengers have the highest retention rates, that is when you purchase an install, they become sticky and typically users won't immediately uninstall.
 
-Not at the moment, the standard we will likely support is [EIP67](https://github.com/ethereum/EIPs/issues/67).
+The average user lifetime sky rockets once their friends are involved. When you start thinking about this as a user base and how to connect that with Ethereum, things like payments and applications built ontop of Ethereum integrated into a converstional interface starts becoming like a clear path forward.
```
Comment (andytudhope) [inline docs/getting-started/faq.md]: This, to me, is literally nonsensical.

**Review by andytudhope — COMMENTED** [review]
Thank you for your effort and @carlbennetts I recommend that we go through this to extract useful FAQs and links, but there are too many grammar and spelling errors, and too wide a discrepancy with the team's core message to merge.

**Comment by jandetlefsen** [comment]
okay i will review the grammar and content.

**Comment by jandetlefsen** [comment]
I've rewritten most content and taken existing content where it fits. Hope this is more useful now

**Comment by pablanopete** [comment]
I am going to save this pages content and pull from it to add to the seperate wiki pages as we update, thanks for the work put into this.

**Pull request closed without merging**